templates/swordSettingsTab.tpl

```diff
@@ -0,0 +1,41 @@
+{**
+ * plugins/generic/sword/templates/swordSettingsTab.tpl
+ *
+ * Copyright (c) 2014-2018 Simon Fraser University
+ * Copyright (c) 2003-2018 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * Deposit articles in remote repositories
+ *}
+
+<tab id="swordSettings" label="Sword Settings"><button class="requestHelpPanel pkp_help_link pkp_help_tab" data-topic="settings" data-section="website">
+	<span class="fa fa-info-circle pkpIcon--inline" aria-hidden="true"></span>
+			Help
```

Comment:
English-language text needs to be moved out to the locale file.

Reply:
@asmecher I couldn't see any content for the Help button to point to, so I just removed it.